vulnerabilities/tests/conftest.py

```diff
@@ -0,0 +1,59 @@
+
+import pytest
+import os
+import json
+
+
+from vulnerabilities.api import PackageSerializer
+from vulnerabilities.data_dump import debian_dump
+from vulnerabilities.data_dump import ubuntu_dump
+from vulnerabilities.data_dump import archlinux_dump
+from vulnerabilities.scraper import archlinux
+from vulnerabilities.scraper import debian
+from vulnerabilities.scraper import ubuntu
+from vulnerabilities.models import Package
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+TEST_DATA = os.path.join(BASE_DIR, 'test_data/')
+
+
+@pytest.fixture
+def setDebianData(db):
+    with open(os.path.join(TEST_DATA, 'debian.json')) as f:
+        test_data = json.load(f)
+
+    extract_data = debian.extract_vulnerabilities(test_data)
+    debian_dump(extract_data)
+
+    Package.objects.create(
+        name='mimetex',
+        version='1.50-1.1',
+        type='deb',
+        namespace='ubuntu'
+    )
+
+
+@pytest.fixture
+def setdDebianData(db):
```

Comment:
This name is *very* easy to confuse with `setDebianData`. :) I think we don't need both functions though. The extra `Package` object that is created in `setDebianData` is only required in one test (`test_debian_query_by_package_url_without_namespace`) to verify that the namespace part of package URLs is optional.

I suggest you delete `setDebianData`, rename `setdDebianData` to `setDebianData` and add the creation of the Ubuntu package to the beginning of `test_debian_query_by_package_url_without_namespace`. That makes it clear that this test data is only needed there. If you do this, you'll also need to adapt some assertions in the other tests that use the current version of `setDebianData`. For example in `test_debian_query_by_name`, four records would be expected in the response instead of five.

Reply:
I had used that for debugging purpose and forgot to remove it. I have removed it now.Sorry